### DISCUSSION TITLE
Using multiple DNS

### DISCUSSION THREAD

**Discussion by http403** [celzero/rethink-app#1004] [Ideas]
I have a self hosted a custom DNS for filtering and I want to use it along side with Tor.

It will be nice to have general DNS queries sent to my custom DNS, and apps with Tor enabled are resolved via Tor.

**Comment by ignoramous** [comment]
`.onion` sent to Orbot / SOCKS5 Tor, yeah? If so, ref:
- #284 
- #421

**Reply by http403** [reply]
No. It is more like a split DNS configuration.

For example
- `.com`, `.net`, `.org` and other general TLDs goes to `doh.example.com`.
- `.lan` goes to `10.0.0.250`
- `.ts.net` goes to `100.100.100.100`
- `.onion` goes to the Tor SOCKS proxy

**Reply by ignoramous** [reply]
Noted: https://github.com/celzero/rethink-app/issues/1040

This is do-able, but the UI for it is going to be a nightmare... A power-user feature, this.
